exercises/practice/error-handling/ErrorHandling.roc

```diff
@@ -0,0 +1,27 @@
+module [getUser, parseUserId, getPage, errorMessage]
+
+User : { name : Str }
+
+users : Dict U64 User
```

Comment:
Using a type alias for `U64` called `UserId` would be more idiomatic, that makes it clear what the U64 represents. It also prevents needing to change every function that uses a UserId if you wanted to change it from U64 to Str for example.

Reply:
I see the benefit of using `UserId`, but I also think it's totally fine to use `U64` directly, so I don't know that introducing a type alias is necessarily more idiomatic.

Reply:
Thanks for your reviews, @Anton-4 & @isaacvando . I like your suggestion @Anton-4 , I just updated the code to add the `UserId : U64` type alias. 👍